app/helpers/is-last-word-in-array.js

```diff
@@ -0,0 +1,5 @@
+import Ember from "ember";
+
+export default Ember.Helper.helper(function(leftside) {
+  return (leftside[1] + 1 < leftside[0]) ? true : false;
```

Comment:
can we write it this way? Do you see any issue? if not that having it in this manner would be good as we are having an expression here
`(leftside[0] > leftside[1] + 1) ? false : true;`